includes/Core/Util/Sanitize.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Util\Sanitize
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2023 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Utility class for sanitizing data.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Sanitize {
+	/**
+	 * Filters empty or non-string elements from a given array.
+	 *
+	 * @since 1.50.0
+	 * @since n.e.x.t Moved the method its own utility class.
```

Comment:
Let's just use @since n.e.x.t` because otherwise it is not clear from where it was moved from.
```suggestion
	 * @since n.e.x.t
```